aws-lambda-scorer/README.md

```diff
@@ -110,6 +110,13 @@ The expected response should follow this structure, but the actual values may di
 
 ```json
 {
+  "id": "a12e7390-b8ac-406a-ade9-0d5ea4b63ea9",
+  "fields": [
```

Comment:
Should we change this to "class" or "classNames" or "targetClasses"?

Reply:
Please read my other comments. This is not only classes and there is also consistency with the input field.

Reply:
agree with @osery here... `fields` encompasses all header values returned, not just the mojo generated classes from target.